### DISCUSSION TITLE
Bug Or Am I Doing Something Wrong?

### DISCUSSION THREAD

**Discussion by cpsmusic** [apohl79/audiogridder#590] [Help] [answered]
Hi All,

I'm new to AudioGridder however I've managed to install and run it on macOS 10.15.7 with Studio One Pro 5. It's amazing! It appears to be working properly however there appears to be a minor problem with the window content.

When I open the AudioGridder plug-in the left side of the window contains the slots for the plug-ins, however the majority of the window grabs the Studio One background and duplicates it (hard to explain so I've got a screenshot for those interested).

<img width="1194" alt="Screen Shot 2021-05-27 at 7 22 42 pm" src="https://user-images.githubusercontent.com/6424857/119803486-d8154d00-bf22-11eb-9415-3f80a0ec025e.png">

Cheers,

Chris

**Comment by apohl79** [comment, marked as answer]
Looks like you want to turn on the local mode in the server settings. Disable screen capturing and enable the local mode checkbox.

**Reply by cpsmusic** [reply]
Thanks, yes that's fixed it. It's an amazing piece of software!
